java/ql/lib/ext/TopJdkApis.model.yml

```diff
@@ -0,0 +1,133 @@
+      - ["java.util", "Optional", False, "ofNullable", "(Object)", "", "Argument[0]", "ReturnValue.Element", "value", "manual"]                  # SUPPORTED: already modelled in java.util.model.yml at line 270, but no signature specified in existing model
+      - ["java.util", "Optional", False, "get", "()", "", "Argument[-1].Element", "ReturnValue", "value", "manual"]                              # SUPPORTED: already modelled in java.util.model.yml at line 264, but no signature specified in existing model
+      - ["java.util", "Optional", False, "ifPresent", "(Consumer)", "", "Argument[-1].Element", "Argument[0].Parameter[0]", "value", "manual"]   # SUPPORTED: already modelled in java.util.model.yml at line 265, but no signature specified in existing model
+      - ["java.util", "Optional", False, "map", "(Function)", "", "Argument[-1].Element", "Argument[0].Parameter[0]", "value", "manual"]         # SUPPORTED: already modelled in java.util.model.yml at line 267, but no signature specified in existing model
+      - ["java.util", "Optional", False, "map", "(Function)", "", "Argument[0].ReturnValue", "ReturnValue.Element", "value", "manual"]           # SUPPORTED: already modelled in java.util.model.yml at line 268, but no signature specified in existing model
+      - ["java.util.stream", "Stream", True, "filter", "(Predicate)", "", "Argument[-1].Element", "Argument[0].Parameter[0]", "value", "manual"] # SUPPORTED: already modelled in java.util.stream.model.yml at line 24
+      - ["java.util.stream", "Stream", True, "filter", "(Predicate)", "", "Argument[-1].Element", "ReturnValue.Element", "value", "manual"]      # SUPPORTED: already modelled in java.util.stream.model.yml at line 25
+      - ["java.util.stream", "Stream", True, "findFirst", "()", "", "Argument[-1].Element", "ReturnValue.Element", "value", "manual"]            # SUPPORTED: already modelled in java.util.stream.model.yml at line 27
+      - ["java.util.stream", "Stream", True, "map", "(Function)", "", "Argument[-1].Element", "Argument[0].Parameter[0]", "value", "manual"]     # SUPPORTED: already modelled in java.util.stream.model.yml at line 45
+      - ["java.util.stream", "Stream", True, "map", "(Function)", "", "Argument[0].ReturnValue", "ReturnValue.Element", "value", "manual"]       # SUPPORTED: already modelled in java.util.stream.model.yml at line 46
+      - ["java.util.stream", "Stream", True, "forEach", "(Consumer)", "", "Argument[-1].Element", "Argument[0].Parameter[0]", "value", "manual"] # SUPPORTED (extra one from Michael's list): already modelled in java.util.stream.model.yml at line 33
+      # - ["java.util.stream", "Stream", True, "collect", "(Collector)", "", "", "", "value", "manual"]                                          # **UNSUPPORTED**: noted in java.util.stream.model.yml at line 19 that "collect(Collector<T,A,R> collector) is handled separately on a case-by-case basis as it is too complex for MaD" (Note: look into if/how this is ever handled)
```

Comment:
Yes, it's handled in https://github.com/github/codeql/blob/main/java/ql/lib/semmle/code/java/frameworks/Stream.qll - at least common cases of it is - it needs per-collector special-casing.